app/assets/stylesheets/velum_general.scss

```diff
@@ -0,0 +1,64 @@
+// Background color for mail container
+$velum-container-bg:          #f0f1f3;
+// Background color for header
+$velum-header-bg:             #00243e;
+
+html,
+body {
+    height: 100%;
+}
+
+.container {
+    background-color: $velum-container-bg;
+    min-height: 100%;
+    position: relative;
+
+    header .col-xs-12 {
+        background-color: $velum-header-bg ;
+
+        .page-header {
+            border-bottom: 0px;
+            height: 60px;
+            margin: 0px 0px;
+            padding: 0px;
+
+            h2 {
+                color: #ffffff;
+                font-size: 18px;
+                margin: 0px;
+                padding-top: 24px;
+            }
+        }
+    }
+
+    .alert {
+        margin-top: $line-height-computed;
+
+        i {
+            bottom: 10px;
+            position: relative;
+        }
+    }
+
+    h1 {
+        margin: 30px 0px;
+    }
+
+    .steps-container .btn-danger {
+        margin-right: $line-height-computed;
+    }
+
+    footer {
+        bottom: 0px;
+        margin-bottom: 15px;
+        position: absolute;
+        width: 100%;
+
+        p {
+            margin: 0px;
+            font-size: 11px;
+        }
+    }
+}
+
+@import 'pages/nodes_list';
```

Comment:
I think is would be simpler if only used `@import` inside the application.scss. It seems to me that this file acts as the "super" css file that should import all the velum css. The application.scss can already take that role. We will only need to extract the rules above in a separate css file (common.scss or somthing like that) and import it in the application.scss. I can take care of that if we agree.